pom.xml

```diff
@@ -78,12 +91,17 @@
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-admin-client</artifactId>
-            <version>20.0.1</version>
+            <version>${keycloak.version}</version>
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-authz-client</artifactId>
-            <version>20.0.1</version>
+            <version>${keycloak.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-properties-migrator</artifactId>
```

Comment:
`spring-boot-properties-migrator` must be used to check in logs deprecated properties, then must be removed

```
2022-12-14T11:59:31.362+02:00  WARN 24760 --- [  restartedMain] org.hibernate.orm.deprecation            : HHH90000021: Encountered deprecated setting [javax.persistence.sharedCache.mode], use [jakarta.persistence.sharedCache.mode] instead
2022-12-14T11:59:31.435+02:00  WARN 24760 --- [  restartedMain] org.hibernate.orm.deprecation            : HHH90000026: MySQL8Dialect has been deprecated; use org.hibernate.dialect.MySQLDialect instead
```

Reply:
This is actually logs not from `spring-boot-properties-migrator`.
For `spring-boot-properties-migrator` you need to run `maven clean package`, and look in logs for `PropertiesMigrationListener`. Seems like you don't have deprecated properties.